syz-ci/manager.go

```diff
@@ -62,21 +62,23 @@ func init() {
 //  - latest: latest known good kernel build
 //  - current: currently used kernel build
 type Manager struct {
-	name       string
-	workDir    string
-	kernelDir  string
-	currentDir string
-	latestDir  string
-	compilerID string
-	configTag  string
-	configData []byte
-	cfg        *Config
-	repo       vcs.Repo
-	mgrcfg     *ManagerConfig
-	managercfg *mgrconfig.Config
-	cmd        *ManagerCmd
-	dash       *dashapi.Dashboard
-	stop       chan struct{}
+	name               string
+	workDir            string
+	kernelDir          string
+	currentDir         string
+	latestDir          string
+	compilerID         string
+	configTag          string
+	configData         []byte
+	baselineConfigTag  string
+	baselineConfigData []byte
```

Comment:
Do we need it here?
This Manager thing does not do bisection. Bisection is done in jobs.go and it will have access to the baseline config via Manager.mgrcfg.KernelBaselineConfig.

Reply:
This one I didn't address in my update to the pull request. I will try to address it in next update. I go errors from compilation if I removed these.

Reply:
This commit looks good to me, except this part.
I would expect it to be easy to resolve. baselineConfigTag does not seem to be used at all, so it can be removed. And the code that reads in baselineConfigData can be moved directly to jobs.go where it is used, that part should have access to the user config.

Reply:
This is now fixed by reading baseline config contents in jobs.go